src/config/section/storage.js

```diff
@@ -168,7 +168,7 @@ export default {
           label: 'label.migrate.volume',
           args: ['volumeid', 'storageid', 'livemigrate'],
           dataView: true,
-          show: (record, store) => { return record.state === 'Ready' && ['Admin'].includes(store.userInfo.roletype) && record.virtualmachineid },
+          show: (record, store) => { return record.state === 'Ready' && ['Admin'].includes(store.userInfo.roletype) },
```

Comment:
@Pearl1594 @davidjumani do we allow migration of attached VMs, for all hypervisors? What's the logic/check in old UI?

Reply:
@rhtyd 
```
if (jsonObj.type == "ROOT" || jsonObj.type == "DATADISK") {
  if (jsonObj.state == "Ready" && isAdmin() && jsonObj.virtualmachineid != null) {
    allowedActions.push("migrateVolume");
  }
}
```